docs/source/code/spatialstats.py

```diff
@@ -0,0 +1,40 @@
+import xdem
```

Comment:
I usually prepend the script with a one-liner docstring to shortly describe the aim with this particular code example.

Also, it doesn't seem to be used yet in the text, right? Same with the other code example.

Reply:
Coming soon!